client/src/components/RatingsAndReviews/Ratings/RatingOverall.jsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-restricted-syntax */
+import React from 'react';
+import ProductStar from '../../Overview/ProductStar';
+
+function round(value, precision) {
+  const multiplier = 10 ** (precision || 0);
+  return Math.round(value * multiplier) / multiplier;
+}
+
+function RatingOverall({ metaData }) {
+  const metaRatings = metaData.ratings;
+  let total = 0;
+  let votesNum = 0;
+
+  // eslint-disable-next-line guard-for-in
+  for (const prop in metaRatings) {
+    const votesN = metaRatings[prop] * 1;
+    total += prop * 1 * votesN;
+    votesNum += votesN;
+  }
```

Comment:
Instead of doing an eslint-disable on a for...in..., I used Object.keys()...forEach()...

```suggestion
  Object.keys(metaRatings).forEach((prop) => {
    const votesN = metaRatings[prop] * 1;
    total += prop * 1 * votesN;
    votesNum += votesN;
  });